linux_files/setup

```diff
@@ -341,7 +358,7 @@ if [[ "$response" =~ ^[yY]$ ]]
 then
     createtmp
     curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -
-    sudo apt install -t testing nodejs build-essential -y 
+    sudo apt install -t testing nodejs npm build-essential -y 
```

Comment:
`npm` really needed? As `npm` should be already included with `nodejs` installation

Reply:
I tried it and it wasn't, but I will double-check.

Reply:
hmm, interesting, it works on my machine... also considering adding `n` back?